src/ILAMB/ConfCO2.py

```diff
@@ -11,6 +11,13 @@
 import pylab as plt
 import numpy as np
 import os
+import copy
+from ILAMB import ccgfilt
```

Comment:
When importing functionality from the package that you are developing, you need to signify that you are importing from **here** and not some other possible installation of ILAMB. This should be:

`from . import ccgfilt`